gguf-py/gguf/constants.py

```diff
@@ -19,77 +19,79 @@
 
 class Keys:
     class General:
-        ARCHITECTURE         = "general.architecture"
+        ARCHITECTURE = "general.architecture"
         QUANTIZATION_VERSION = "general.quantization_version"
-        ALIGNMENT            = "general.alignment"
-        NAME                 = "general.name"
-        AUTHOR               = "general.author"
-        VERSION              = "general.version"
-        URL                  = "general.url"
-        DESCRIPTION          = "general.description"
-        LICENSE              = "general.license"
-        SOURCE_URL           = "general.source.url"
-        SOURCE_HF_REPO       = "general.source.huggingface.repository"
-        FILE_TYPE            = "general.file_type"
+        ALIGNMENT = "general.alignment"
+        NAME = "general.name"
+        AUTHOR = "general.author"
+        VERSION = "general.version"
+        URL = "general.url"
+        DESCRIPTION = "general.description"
+        LICENSE = "general.license"
+        SOURCE_URL = "general.source.url"
+        SOURCE_HF_REPO = "general.source.huggingface.repository"
+        FILE_TYPE = "general.file_type"
```

Comment:
Don't change the whitespaces like this - these are intentionally aligned vertically

Reply:
It seems that all the python files have been auto-formatted.

Reply:
Removed the auto formatting, sorry about that !